lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -2101,6 +2101,32 @@ class DecomposeAtenNumpyTOp : public OpRewritePattern<AtenNumpyTOp> {
 };
 } // namespace
 
+// Decompose aten.norm.ScalarOpt_dim
+namespace {
+class DecomposeAtenNormScalarOptDimOp
+    : public OpRewritePattern<AtenNormScalarOptDimOp> {
+public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(AtenNormScalarOptDimOp op,
+                                PatternRewriter &rewriter) const override {
+    Location loc = op.getLoc();
+    Value self = op.self();
+    Value p = op.p();
+    Value dim = op.dim();
+    Value keepdim = op.keepdim();
```

Comment:
These two are currently not used in the decomposition. You should be able to incorporate them by using the `AtenSumDimIntListOp` instead of `AtenSumOp`